lib/resources/card.js

```diff
@@ -68,13 +69,14 @@ card.save = card.create;
 
 /**
  * Update card from customer id
- * [Click here for more infos]{@link https://www.mercadopago.com/developers/en/reference/cards/_customers_customer_id_cards_id/put/}
+ * [Click here for more infos]{@link https://www.mercadopago.com.br/developers/en/reference/cards/_customers_customer_id_cards_id/put/}
```

Comment:
All documentation is using the .com domain. If there's no specific reason to add BR, we'd better just keep .com

Reply:
Done!